docs/concepts.rst

```diff
@@ -415,13 +415,31 @@ explictly specify a serialization strategy for an entity by attaching a
 
 .. _Protocol: api/protocols.rst
 
-Exporting Persisted Files
-.........................
+Retrieving Persisted Files
+..........................
+In some cases, you'll want to directly access the persisted file(s) for an
+entity rather than its in-memory representation.  (For example, if you're
+writing a paper or report, you may want to access the files containing the
+plots.) This can be achieved using the :meth:`Flow.get <bionic.Flow.get>`
+method. Its ``collection`` parameter specifies whether to return a single value
+or, if more than one, the wrapping data structure.  Its ``mode`` parameter
+indicates the type of object to represent each persisted file.  For example:
+
+.. code-block:: python
+
+    flow = builder.build().setting('subject', values=['Alice', 'Bob'])
+    flow.get('subject', collection=list, mode=Path)
+
+This would return a ``List[Path]``, one ``Path`` per persisted file (in this case, 2).
+
+We can also take advantage of the :class:`FileCopier <bionic.util.FileCopier>` class:
+
+.. code-block:: python
 
-In some cases, you'll want to directly access the persisted file for an entity,
-rather than its in-memory representation.  (For example, if you're writing a
-paper or report, you may want to access the files containing the plots.) This
-can be accomplished using the :meth:`Flow.export <bionic.Flow.export>` method.
+    file_copiers = flow.get('subject', collection=list, mode='FileCopier')
+    for file_copier in file_copiers:
+        # copy all of the entity's files to the current working dir
```

Comment:
Super super nit: Just to be consistent with the rest of the examples in these docs, I'd capitalize the first character and add a period at the end.